### PR TITLE
INTPYTHON-406 Add automated release workflows for Django-MongoDB

### PR DESCRIPTION
**Pull request by blink1073**
Depends on https://github.com/mongodb-labs/drivers-github-tools/pull/61

### CODE REVIEW

**Comment by aclark4life** [comment]
Thanks @blink1073! Does this mean we can test release to Test PyPI when merged?

File: django_mongodb/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = "5.0a0"
+__version__ = "5.0.0.dev2"
```
[the remark this answers is not in the document]
Comment (aclark4life) [reply django_mongodb/__init__.py:1]: `devX` [seems to be a convention](https://github.com/python-pillow/Pillow/commits/main/src/PIL/_version.py?before=0995305eb4f991c71e157a04c731af7c7ee9785c+35) but I'm not sure what it indicates … we are on `dev2` now apparently 😄

[the remark this answers is not in the document]
Comment (blink1073) [reply django_mongodb/__init__.py:1]: It was auto-incremented by the releaser, I'll switch back to 5.0a0

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+django>=5.0,<5.1
```
[the remark this answers is not in the document]
Comment (blink1073) [reply requirements.txt:2]: Done

**Review by Jibola — APPROVED** [review]

**Pull request merged**